specification/webpubsub/SignalRService.Management/models.tsp

```diff
@@ -0,0 +1,1612 @@
+   */
+  unit?: string;
+}
+
+/**
+ * Localizable String object containing the name and a localized value.
+ */
+#suppress "@azure-tools/typespec-azure-core/casing-style" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+model SignalRServiceUsageName {
+  /**
+   * The identifier of the usage.
+   */
+  value?: string;
+
+  /**
+   * Localized name of the usage.
+   */
+  localizedValue?: string;
+}
+
+/**
+ * Object that includes an array of resources and a possible link for next set.
+ */
+model WebPubSubResourceList is Azure.Core.Page<WebPubSubResource>;
+
+/**
+ * The billing information of the resource.
+ */
+model ResourceSku {
+  /**
+   * The name of the SKU. Required.
+   *
+   * Allowed values: Standard_S1, Free_F1, Premium_P1, Premium_P2
+   */
+  name: string;
+
+  /**
+   * Optional tier of this particular SKU. 'Standard' or 'Free'.
+   *
+   * `Basic` is deprecated, use `Standard` instead.
+   */
+  tier?: WebPubSubSkuTier;
+
+  /**
+   * Not used. Retained for future use.
+   */
+  @visibility(Lifecycle.Read)
+  size?: string;
+
+  /**
+   * Not used. Retained for future use.
+   */
+  @visibility(Lifecycle.Read)
+  family?: string;
+
+  /**
+   * Optional, integer. The unit count of the resource.
+   * 1 for Free_F1/Standard_S1/Premium_P1, 100 for Premium_P2 by default.
+   *
+   * If present, following values are allowed:
+   *     Free_F1: 1;
+   *     Standard_S1: 1,2,3,4,5,6,7,8,9,10,20,30,40,50,60,70,80,90,100;
+   *     Premium_P1:  1,2,3,4,5,6,7,8,9,10,20,30,40,50,60,70,80,90,100;
+   *     Premium_P2:  100,200,300,400,500,600,700,800,900,1000;
+   */
+  capacity?: int32;
+}
+
+/**
+ * A class that describes the properties of the resource
+ */
+model WebPubSubProperties {
+  /**
+   * Provisioning state of the resource.
+   */
+  @visibility(Lifecycle.Read)
+  provisioningState?: ProvisioningState;
+
+  /**
+   * The publicly accessible IP of the resource.
+   */
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @visibility(Lifecycle.Read)
+  externalIP?: string;
+
+  /**
+   * FQDN of the service instance.
+   */
+  @visibility(Lifecycle.Read)
+  hostName?: string;
+
+  /**
+   * The publicly accessible port of the resource which is designed for browser/client side usage.
+   */
+  @visibility(Lifecycle.Read)
+  publicPort?: int32;
+
+  /**
+   * The publicly accessible port of the resource which is designed for customer server side usage.
+   */
+  @visibility(Lifecycle.Read)
+  serverPort?: int32;
+
+  /**
+   * Version of the resource. Probably you need the same or higher version of client SDKs.
+   */
+  @visibility(Lifecycle.Read)
+  version?: string;
+
+  /**
+   * Private endpoint connections to the resource.
+   */
+  @visibility(Lifecycle.Read)
+  @OpenAPI.extension("x-ms-identifiers", #[])
+  privateEndpointConnections?: PrivateEndpointConnection[];
```

Comment:
We got complaint about [`invalid-ref`s](https://dev.azure.com/azure-sdk/public/_build/results?buildId=5053857&view=logs&j=83516c17-6666-5250-abde-63983ce72a49&t=00be4b52-4a63-5865-8e02-c61723ad0692&l=354).
It happens for all SDK emitters.

The tsp structure seem weird. 
In `models.tsp`, it referenced `PrivateEndpointConnection` from `PrivateEndpointConnectionResource.tsp`, but it does not import the file. and in `PrivateEndpointConnectionResource.tsp`, it referenced `models.tsp`..

Maybe we need to restructure these models. E.g. put these `XXProperties` into corresponding `XX.tsp`. E.g. `WebPubSubProperties` should go into `WebPubSubResource.tsp` instead of `models.tsp`.